File: CS/BarCode/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors.Controls;
using DevExpress.XtraEditors;
using DevExpress.XtraPrinting.BarCode;
using DevExpress.XtraPrinting.BarCode.Native;

namespace BarCode {
    public partial class Form1 : Form {

        public Form1() {
            InitializeComponent();
            CalcRadioGroup(typeof(BarCodeSymbology), radioGroup1);
        }

        private void CalcRadioGroup(Type EnumType, RadioGroup radioGroup) {
            foreach (object item in Enum.GetValues(EnumType)) {
                RadioGroupItem radioItem = new RadioGroupItem(item, item.ToString());
                radioGroup.Properties.Items.Add(radioItem);
            }
        }

        private void radioGroup1_SelectedIndexChanged(object sender, EventArgs e) {
            barCodeControl1.Symbology = BarCodeGeneratorFactory.Create((BarCodeSymbology)radioGroup1.EditValue);
        }
    }
}

File: VB/BarCode/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors.Controls
Imports DevExpress.XtraEditors
Imports DevExpress.XtraPrinting.BarCode
Imports DevExpress.XtraPrinting.BarCode.Native

Namespace BarCode
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			CalcRadioGroup(GetType(BarCodeSymbology), radioGroup1)
		End Sub

		Private Sub CalcRadioGroup(ByVal EnumType As Type, ByVal radioGroup As RadioGroup)
			For Each item As Object In System.Enum.GetValues(EnumType)
				Dim radioItem As New RadioGroupItem(item, item.ToString())
				radioGroup.Properties.Items.Add(radioItem)
			Next item
		End Sub

		Private Sub radioGroup1_SelectedIndexChanged(ByVal sender As Object, ByVal e As EventArgs) Handles radioGroup1.SelectedIndexChanged
			barCodeControl1.Symbology = BarCodeGeneratorFactory.Create(CType(radioGroup1.EditValue, BarCodeSymbology))
		End Sub
	End Class
End Namespace
